classes/PodsForm.php

```diff
@@ -825,68 +825,68 @@ public static function field_setup( $field = null, $core_defaults = null, $type
      * @param array $options array( 'depends-on' => ..., 'excludes-on' => ...)
      * @param string $prefix
      *
-     * @return string
+     * @return array
```

Comment:
The changes in this commit were in preparation for some more changes to be made but that turned out to be unnecessary.